web/EXAMPLES.md

```diff
@@ -130,30 +135,49 @@ The second rule is limited to blocking all matching requests except for those wh
 | example.com | test-tracker.net/ddm/ | image  |  false | matches rule and matches rule exception |
 | example.com | test-tracker.net/adimage.png | image  |  true | no matching rule, default is set to 'block' |
 
-### Rule with `exceptions`
-
-We have `default: ignore`, so we only block requests that match a rule. The first rule doesn't have any `exceptions`, so we block any matching request. The second rule has a blocking exception. We will not block matching `image` requests that come from `test-site-2.com` or its subdomains.
+### Rule with `exceptions` and `options`
 
+We have `default: ignore`, so we only block requests that match a rule. 
+- The first rule doesn't have any `options` or `exceptions`, so we block any matching request.
+- The second rule has `options`, we will only block requests that match the rule and come from a site matching the domains list.
+- The third rule has a both `exceptions` and `options`. For this rule we only block the request when it matches one of the `options` domains, and only if it doesn't also match an `exceptions` domain. 
 ```json
 {
     "domain": "example.net",
+    "default": "ignore",
     "rules": [
         {
             "rule": "connect\\.example\\.net\\/signals\\/"
         },
+	{
+	    "rule": "sometimes-tracking\\.example\\.net",
+	    "options": {
+	    	"domains": [
+		    "test-site-3.com"
+		]
+	    }
+	},
         {
             "rule": "example\\.net\\/.*\\/AudienceNetworkVPAID\\.",
+	    "options": {
+	    	"domains": [
+		    "test-site-2.com",
+		    "test-site-3.com"
+		], 
+		"types": [
+	            "script"
+		]
+	    },
 	    "exceptions": {
 	    	"types": [
-		    "image"
+		    "script"
```

Comment:
it's slightly confusing that we have same combination in both options and exceptions (test-site-2 + script). Was this on purpose?